app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationScreen.kt

```diff
@@ -76,8 +83,16 @@ fun CompleteRegistrationScreen(
                 Toast.makeText(context, event.message(context.resources), Toast.LENGTH_SHORT).show()
             }
 
-            is CompleteRegistrationEvent.NavigateToLanding -> {
-                onNavigateToLanding()
+            CompleteRegistrationEvent.NavigateToMakePasswordStrong -> onNavigateToPasswordGuidance()
+            CompleteRegistrationEvent.NavigateToPreventAccountLockout -> {
+                onNavigateToPreventAccountLockout()
+            }
+
+            is CompleteRegistrationEvent.NavigateToLogin -> {
+                onNavigateToLogin(
+                    event.email,
+                    event.captchaToken,
```

Comment:
Wanna add some names

Reply:
I can add commented ones java style, doesn't allow call names on function typed

Reply:
added on  line 73 : `onNavigateToLogin: (email: String, token: String) -> Unit,` so order is defined somewhere 👍

Reply:
🤦 

Don't worry about it.